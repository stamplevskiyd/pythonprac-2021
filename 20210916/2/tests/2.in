[2,5,1,12345]
