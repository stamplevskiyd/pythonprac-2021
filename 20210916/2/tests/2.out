[1, 2, 5, 12345]
